pages/_app.tsx

```diff
@@ -70,8 +81,20 @@ CustomApp.getInitialProps = async (
       appContext.ctx.res.end();
     }
   }
+
   const appProps = await App.getInitialProps(appContext);
-  return { ...appProps, pageProps: { ...appProps.pageProps, user } };
+
+  const environmentName = [
+    'social-care-service-staging.hackney.gov.uk',
+    'dev.hackney.gov.uk:3000',
+  ].includes(process.env.REDIRECT_URL || '')
```

Comment:
What would `.includes(process.env.REDIRECT_URL || '')` help us do?

Reply:
This is effectively saying: "if the value of `REDIRECT_URL` is one of our known development URLs, then set the environment name to "development". This is intentionally defensive, so that if we happened to change our production URL, this feature flag wouldn't accidentally be activated.